src/OptionSettlementEngine.sol

```diff
@@ -103,43 +103,43 @@ contract OptionSettlementEngine is ERC1155, IOptionSettlementEngine {
 
     /// @inheritdoc IOptionSettlementEngine
     function option(uint256 tokenId) external view returns (Option memory optionInfo) {
-        (uint160 optionKey,) = decodeTokenId(tokenId);
-        optionInfo = optionRecords[optionKey].option;
+        (uint160 optionNum,) = decodeTokenId(tokenId);
```

Comment:
optionNum and claimNum over optionKey and claimKey? Just wondering thoughts, I'm kinda liking key over num but both work and either work to disambiguate from tokenId/optionId/claimId